source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -241,29 +278,29 @@ A driver-defined entity that encapsulates all non-monitoring connections associa
       generation: number;
     
       /**
-       *  An integer expressing how many total connections
+       *  An integer expressing how many total Connections
        *  (active + in use) the pool currently has
```

Comment:
"active" is now not a defined state, I think this should say "pending" + "available" + "in use".